runtime/common/src/crowdloan/mod.rs

```diff
@@ -145,33 +147,33 @@ pub enum LastContribution<BlockNumber> {
 #[codec(dumb_trait_bound)]
 pub struct FundInfo<AccountId, Balance, BlockNumber, LeasePeriod> {
 	/// The owning account who placed the deposit.
-	depositor: AccountId,
+	pub depositor: AccountId,
```

Comment:
nit: looks like we can limit the scope here to `pub(crate)`

Reply:
meh, people may make pallets which use this, and ask us for access to it anyway. i am opting for this to be full pub